app/controllers/api/pictures_controller.rb

```diff
@@ -1,11 +1,13 @@
 module Api
   class PicturesController < BaseController
     def create_resource(_type, _id, data)
-      data['content'] = Base64.decode64(data['content'])
-      picture = Picture.create(data)
-      raise BadRequestError,
-            "Failed to create Picture - #{picture.errors.full_messages.join(', ')}" unless picture.valid?
-      picture
+      raise 'requires content' unless data['content']
+      Picture.new(data.except('content')).tap do |picture|
+        picture.content = Base64.strict_decode64(data['content'])
```

Comment:
@imtayadeway strict_decode64 still doesn't like nils, so I had to keep in the `requires content` validation